editorconfig-eclipse-plugin/editorconfig-core-java

```diff
@@ -1 +0,0 @@
-Subproject commit 9ff6e78159540f5c305551237fcf2c7f14701b1d
```

Comment:
@ncjones I don't know how to that -( And I have not found how I could recreate this subproject. Please help me.